drivers/ieee802154/ieee802154_rf2xx.c

```diff
@@ -53,10 +53,7 @@ LOG_MODULE_REGISTER(LOG_MODULE_NAME);
 
 static uint8_t rf2xx_ack_psdu[RF2XX_ACK_FRAME_LEN] = { 0 };
 static struct net_buf rf2xx_ack_frame = {
-	.data  = rf2xx_ack_psdu,
-	.size  = RF2XX_ACK_FRAME_LEN,
-	.len   = RF2XX_ACK_FRAME_LEN,
```

Comment:
Not sure if it's a problem, but the initializer will now set the `len` field to 0 - @nandojve what's your say?

Reply:
Hi @rlubos ,

This exists only to send ACK to OpenThread. At rf2xx driver, it is applied same principle of `create_ack_frame` test method:
https://github.com/zephyrproject-rtos/zephyr/blob/2130e2b549e2f73fe99dfff030a2f25aff717396/tests/subsys/openthread/radio_test.c#L389-L408

The big difference is that at rf2xx the packet is static allocated. If data should be consumed at `ieee802154_radio_handle_ack`, I think it will be necessary explicit set `.len = 0` before call it, at:

https://github.com/zephyrproject-rtos/zephyr/blob/2130e2b549e2f73fe99dfff030a2f25aff717396/drivers/ieee802154/ieee802154_rf2xx.c#L524-L526

I'm not complete aware of `ieee802154_radio_handle_ack` but that maybe necessary to avoid send this fragment to upper layers.